frontend/src/components/molecules/HighlightWithTwoButtons/style.css

```diff
@@ -0,0 +1,14 @@
+.datapass-container-two-btn {
+  display: flex;
+  justify-content: space-between;
+  gap: 1rem;
+  padding-bottom: 5rem;
+  border-bottom: 1px solid var(--border-default-grey);
```

Comment:
this border is a delimiter in the HeadSection component. This should be put there with its couterpart on top of the section.

Reply:
I remove the style from HeadSection/index.js file to put in a css file. 

<img width="978" alt="Capture d’écran 2022-08-05 à 17 25 07" src="https://user-images.githubusercontent.com/43042737/183109638-e5124908-f6fa-42bd-a018-fe25d791845d.png">